Tests/LibWeb/Text/expected/form-formMethod-attribute.txt

```diff
@@ -0,0 +1,103 @@
+button: unset
+button.getAttribute('formMethod') == 'null'
+button.formMethod == 'get'
```

Comment:
This is wrong the expected value is an empty string

Reply:
This might be due to https://github.com/LadybirdBrowser/ladybird/blob/master/Userland/Libraries/LibWeb/HTML/HTMLFormElement.cpp#L417-L437

Reply:
I don't think this, is used while getting the formMethod IDL value, the getter for it is implemented in the IDL generated code, at least to my understanding